apps/server/src/main.ts

```diff
@@ -1,11 +1,33 @@
-import express = require('express');
+import express from 'express';
+import http from 'http';
+import { Server } from 'socket.io';
 const app = express();
 const port = process.env.PORT || 3000;
+const server = http.createServer(app);
 
 app.get('/', (req, res) => {
   res.send('Hello World!');
 });
 
-app.listen(port, () => {
-  console.log(`App listening on port ${port}`);
+const allowedOrigin =
+  process.env.NX_ALLOWED_ORIGINS.split(', ') || 'http://localhost:4200';
+
+const io = new Server(server, {
+  cors: {
```

Comment:
@dalanad In here I'm using `cors` package to configure the `Access-Control-Allow-Origin`

Reply:
doesn't it automatically get installed with socket.io?

Reply:
@supunTE

Reply:
![image](https://user-images.githubusercontent.com/33794760/223947938-b71a1648-0c65-4ac5-bf2a-f205a7a33ddb.png)
https://socket.io/docs/v3/handling-cors/

Reply:
removed the package d725ceb

Reply:
Yeah, we don't need cors to install manually. socket IO is providing us with that. I misunderstood the issue.
removed the package https://github.com/the-ai-team/razor/commit/d725ceb42b18d42d53f02fb0d09390fcb0cea0ae